<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskQuery</class>
 <widget class="QDialog" name="taskQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>291</width>
     <height>224</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>제목</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="LblEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="deadline">
      <property name="text">
       <string>제출기한</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="DateEdit">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
      <property name="date">
       <date>
        <year>2020</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="require">
      <property name="text">
       <string>필요 시간</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="type">
      <property name="text">
       <string>분류</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="TypeBox">
      <item>
       <property name="text">
        <string>수행평가</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>사이드 프로젝트</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>대회</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>기타: </string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="co_work">
      <property name="text">
       <string>팀과제</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="Co_Work"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="description">
      <property name="text">
       <string>설명</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPlainTextEdit" name="DescriptionEdit">
      <property name="plainText">
       <string>문제수/중요도/연계성 등
기타 설명
</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTimeEdit" name="TimeEdit">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2020</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2020</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="maximumTime">
       <time>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
       </time>
      </property>
      <property name="minimumTime">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::HourSection</enum>
      </property>
      <property name="displayFormat">
       <string>h:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="time">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>taskQuery</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>taskQuery</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
